config/config.js

```diff
@@ -0,0 +1,29 @@
+function getMergedDataFilename() {
+  const prefix = process.env.NODE_ENV === 'production' ? '' : 'dev-';
```

Comment:
The environment check was removed in the sexual-health-information-etl by allowing the filename to be specified as an env var, and having it set with a `dev-` prefix in the local compose file. Might be worth doing the same here.